pilot/lib/pilot/tester_manager.rb

```diff
@@ -92,26 +93,30 @@ def list_testers(options)
 
     # private
```

Comment:
Looks methods below this can indeed be private. Is that something we want to enforce by uncommenting this line?